platform/commonUI/edit/src/creation/CreateAction.js

```diff
@@ -71,6 +71,12 @@ define(
                 openmct.editor.cancel();
             }
 
+            function isFirstViewEditable(domainObject) {
```

Comment:
We need to check more than just the first view. It will probably work most of the time, but it's not future-proof. If the first view is not editable, `ObjectView.vue` will jump into the first editable view when edit mode is triggered.

```JavaScript
let hasEditableView = openmct.objectViews.get(domainObject)
    .some(view => view.canEdit && view.canEdit(domainObject));
```

Reply:
@akhenry - Going to leave as is, after our conversation today